CHANGELOG.md

```diff
@@ -12,5 +12,5 @@
 
 ## Enhancements: ##
 
-- Add support for migrating partitioned tables.  
+- Add support for migrating partitioned tables.
```

Comment:
Sorry, I forgot to explicitly mention that instance as well.
Just fix it with the documentation patch.

Reply:
Removed. This was only result of my C linter. Look like for this project I should exclude `.md` from linter "no trailing spaces" rule.